aci/resource_aci_fvnsmcastaddrblk.go

```diff
@@ -0,0 +1,219 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAciMulticastAddressBlock() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAciMulticastAddressBlockCreate,
+		UpdateContext: resourceAciMulticastAddressBlockUpdate,
+		ReadContext:   resourceAciMulticastAddressBlockRead,
+		DeleteContext: resourceAciMulticastAddressBlockDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciMulticastAddressBlockImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"multicast_pool_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
ForceNew may not be required here.

Reply:
tested and get errors otherwise, so looks like it is required